AcademyApp/Model/Lecture.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  MockData.swift
+//  AcademyApp
+//
+//  Created by Peter Sevcik on 28/06/2019.
+//  Copyright © 2019 Peter Sevcik. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+struct Lecture {
```

Comment:
comments what data struct it is, is default value for string necessary?

Reply:
Added comment above Lecture struct. Removed default value for string.